openshift/scripts/common/envars

```diff
@@ -8,3 +8,5 @@ PROJ_PROD="${PROJ_PROD:-auzhsi-prod}"
 TAG_PROD="${TAG_PROD:-prod}"
 PATH_BC="${PATH_BC:-$(dirname ${0})/../templates/build.bc.yaml}"
 PATH_DC="${PATH_DC:-$(dirname ${0})/../templates/deploy.dc.yaml}"
+KEYCLOAK_PROD_URL="https://sso.pathfinder.gov.bc.ca/auth"
```

Comment:
@Nkuruba @Kyubinhan we're being silly (re. our discussion about this yesterday) - and sorry if I missed the plot yesterday - but we don't need this at all. The DC should be pulling this from a configmap - so we don't have to specify the url here at all.